packages/app/src/components/AppNotificationManager/notification.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { StyledNotification } from './styled';
+import ContentMetaResolver from './contentMetaResolver';
+
+const Notification = (props) => {
+    const { notification, timeout } = props;
+    const [shouldExit, setShouldExit] = useState(false);
+    const [contentMeta, updateContentMeta] = useState(false);
+    const [shown, setShown] = useState(false);
+
+    if (timeout) {
+        setTimeout(() => {
+            setShouldExit(true);
+        }, timeout);
+    }
+
+    if (contentMeta) {
+        const { height } = contentMeta;
+        setTimeout(() => {
+            setShown(true);
+        }, 0);
```

Comment:
The idea behind the show status is to update the component after it has rendered the first time, hence the timeout. This is necessary because we need to render the notification outside of the view, and then apply change the shown state, so the StyledComponent updates the styles and runs the transitions. That is why after the content meta is resolved, the notification is rendered with shown=false, and then updates to shown=true